x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/legend.tsx

```diff
@@ -122,6 +122,21 @@ export const JobMapLegend: FC<{ theme: EuiThemeType }> = ({ theme }) => {
           </EuiFlexItem>
         </EuiFlexGroup>
       </EuiFlexItem>
+      <EuiFlexItem grow={false}>
```

Comment:
When you render the legend, do you know what type of nodes are visible in the map? If so would be good to only show this legend item when there are missing jobs.

Reply:
Good call - updated to only show that in the legend when there is a matching node in b65e50782fd9bd90e1ef49f5de3f9b1aaed495bd